### ISSUE TITLE
Customize author's nick in webhook

### ISSUE THREAD

**Issue by DarthGandalf** [reactiflux/discord-irc#572]
E.g. make it `... (via IRC)` to make it clear where the message comes from instead of just saying "Bot"

The "format.discord" option from config isn't used in the webhook mode, from what I can see in https://github.com/reactiflux/discord-irc/blob/e0f2edf2d487603db1ee34f847ae7e234c651ff9/lib/bot.js#L597-L602

**Comment by Throne3d** [comment]
I don't believe we can change the "Bot" tag – that comes from Discord. You're right, though, that we don't use the `format.discord` option from the config with webhooks. I think it'd be best to make a new config, `format.webhookName`, if we were to introduce this, as `format.discord` is used to format whole messages (not just usernames) when it's applied.

This project is mostly in maintenance-only mode, so I won't be implementing this myself, but I'll mark it with a "new feature" and "help wanted" tag in case anyone wants to take it up. With an especially complete PR (including tests), I could merge a change to add this!
